dev/find-dead-code.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 cd "$(dirname "$0")"
-cd ..
+cd .. || exit
```

Comment:
Why do we need `|| exit` here and below? Would it be cleaner to not call the scripts in the first place?

Reply:
Issue will be rectified locally only.